front/app/containers/Admin/users/messages.ts

```diff
@@ -70,6 +70,10 @@ export default defineMessages({
     defaultMessage:
       'Are you sure you want to give {name} platform admin rights?',
   },
+  buyAditionalSeat: {
+    id: 'app.containers.Admin.Users.buyAditionalSeat',
+    defaultMessage: 'Buy 1 additional seat',
```

Comment:
```suggestion
    id: 'app.containers.Admin.Users.buyAdditionalSeat',
    defaultMessage: 'Buy one additional seat',
```

Reply:
Small numbers should rather be written in full.

Reply:
Indeed. As a process improvement, could we make copy related suggestions in Figma and update there as-well such that the product and designs are more in line? It also helps limit any back and fourth.

Reply:
Yes, I'll do that in the future.